components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -314,7 +315,7 @@ void P3Microphysics::run_impl (const Real dt)
   // advance it, updating the p3 fields.
   auto ts = timestamp();
   ts += dt;
-  for (auto& f : m_p3_fields_out) {
+  for (auto& f : m_fields_out) {
```

Comment:
TODO: this seems like another good candidate for stuff to move in the base class, since we kind of do it in all subclasses.